src/main/java/com/amazon/opendistroforelasticsearch/sql/exception/SqlFeatureNotImplementedException.java

```diff
@@ -0,0 +1,19 @@
+package com.amazon.opendistroforelasticsearch.sql.exception;
+
+/**
+ * Intended for cases when we knowingly omitted some case, letting users know that we didn't implemented feature, but
+ * it may be implemented in future.
+ */
+public class SqlFeatureNotImplementedException extends RuntimeException {
+    static final long serialVersionUID = 1;
+
+    public SqlFeatureNotImplementedException() { }
```

Comment:
I think this should call the base constructor with some default message.

Reply:
It seems like typical pattern is just to call super() there, fixed